lib/Driver/Driver.cpp

```diff
@@ -1814,10 +1815,17 @@ static StringRef getOutputFilename(Compilation &C,
       BaseName = llvm::sys::path::stem(BaseInput);
     if (auto link = dyn_cast<LinkJobAction>(JA)) {
       if (link->getKind() == LinkKind::DynamicLibrary) {
-        // FIXME: This should be target-specific.
-        Buffer = "lib";
+        if (Triple.isOSWindows())
```

Comment:
Drive by food for thought: Why not hoist this into LLVM? How does clang do this?

Reply:
Oh, absolutely, I felt the same! I didn't look where the best place for this to live would be, though. I'll check what Clang does and comment back here.